frontend/src/store/index.js

```diff
@@ -25,6 +25,144 @@ Vue.use(Vuex)
 export const store = new Vuex.Store({
 
   state: {
+    addIndividualITAsteps: [
+      {
+        step: 1,
+        title:'Exam Type',
+        questions: [
+          {
+            key: 'exam_type_id',
+            text: 'Exam Type ID / Colour',
+            kind:'dropdown',
+            minLength: 0,
+            digit: false,
+          }
+        ]
+      },
+      {
+        step: 2,
+        title:'Exam Info',
+        questions: [
+          {
+            key: 'event_id',
+            text:'Event ID' ,
+            kind: 'input',
+            minLength: 6,
+            digit: true,
+          },
+          {
+            key: 'exam_name',
+            text: 'Exam Name',
+            kind: 'input',
+            minLength: 6,
+            digit: false
+          },
+          {
+            key: 'examinee_name',
+            text: `Exam Writer's Name`,
+            minLength: 6,
+            kind:'input',
+            digit: false
+          },
+          {
+            key: 'exam_method',
+            text: 'Exam Method',
+            minLength: 0,
+            digit: false,
+            kind:'select',
+            options: [
+              {text: 'paper', value: 'paper', id: 'exam_method'},
```

Comment:
Can paper be the default option when the form loads?

Reply:
it can.  Would online not be the more frequent option though if we are going for sensible, time-saving default values?

Reply:
I think paper is the more frequently used option. I don't actually know though